subsys/fs/littlefs_fs.c

```diff
@@ -57,6 +59,16 @@ BUILD_ASSERT((CONFIG_FS_LITTLEFS_HEAP_PER_ALLOC_OVERHEAD_SIZE % 8) == 0);
 
 static K_HEAP_DEFINE(file_cache_heap, CONFIG_FS_LITTLEFS_FC_HEAP_SIZE);
 
+static inline bool littlefs_on_blkdev(struct fs_mount_t *mountp)
+{
+	if (IS_ENABLED(CONFIG_FS_LITTLEFS_BLK_DEV) &&
+	    mountp->flags & FS_MOUNT_FLAG_USE_DISK_ACCESS) {
+		return true;
+	}
+
+	return false;
```

Comment:
Btw. this code could just be wrapped to
```
return (IS_ENABLED(CONFIG_FS_LITTLEFS_BLK_DEV) && (mountp->flags & FS_MOUNT_FLAG_USE_DISK_ACCESS));
```

Reply:
That is really ugly. The following is clean, easy to understand, ...
```
static inline bool littlefs_on_blkdev(struct fs_mount_t *mountp)
{
	if (!IS_ENABLED(CONFIG_FS_LITTLEFS_BLK_DEV)) {
		return false;
	}

	return mountp->flags & FS_MOUNT_FLAG_USE_DISK_ACCESS;
}
```

Reply:
Ok.